wasmd/cmd/wasmd.go

```diff
@@ -0,0 +1,73 @@
+package cmd
+
+import (
+	"errors"
+
+	"github.com/ignite/cli/ignite/pkg/cliui"
+	"github.com/ignite/cli/ignite/pkg/xgit"
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+)
+
+const (
+	statusWasmInit = "Adding Wasm to your chain..."
+)
+
+// NewWasmd creates a new wasmd command that holds
+// some other sub commands related to running chain wasmds like gex and etc.
```

Comment:
Maybe something like?

```suggestion
// NewWasmd creates a new wasmd command that holds
// some other sub commands related to cosmwasm.
```